WalletWasabi.Fluent/ViewModels/Wallets/Home/Tiles/WalletBalanceTileViewModel.cs

```diff
@@ -15,15 +24,46 @@ public WalletBalanceTileViewModel(WalletViewModel walletVm)
 			.Select(_ => wallet.Coins.TotalAmount());
 
 		BalanceBtc = balance
-			.Select(money => $"{money.ToFormattedString()} BTC");
+			.Select(FormatMoney);
 
 		BalanceFiat = balance
 			.Select(money => money.BtcToUsd(wallet.Synchronizer.UsdExchangeRate));
 
 		HasBalance = balance
 			.Select(money => money > Money.Zero);
+
+		CopyBalanceCommand = ReactiveCommand.CreateFromObservable<PointerEventArgs, Unit>(eventArgs =>
+		{
+			if (!eventArgs.GetCurrentPoint((IVisual?) eventArgs.Source).Properties.IsRightButtonPressed)
+			{
+				return Observable.Empty(Unit.Default);
+			}
+
+			return Observable.FromAsync(() => SetTextAsync(wallet))
+				.ToSignal()
+				.Concat(Observable.Timer(TimeSpan.FromSeconds(2)).ToSignal());
```

Comment:
IMO we better decrease the time that the `copied` label is displayed.

```suggestion
				.Concat(Observable.Timer(TimeSpan.FromSeconds(1)).ToSignal());
```